package.json

```diff
@@ -7,7 +7,9 @@
   "author": "bzabos <b.zabos@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "tsc --declaration; ./node_modules/.bin/ts-json-schema-generator --type IContainer --tsconfig 'tsconfig.json' --out 'dist/resources/flowSpecJsonSchema.json'",
+    "jsonschema:generate-container": "./node_modules/.bin/ts-json-schema-generator --type IContainer --tsconfig 'tsconfig.json' --out 'dist/resources/flowSpecJsonSchema.json'",
```

Comment:
@bzabos , I'd propose to change the output file name for IContainer, `flowSpecJsonSchema.json` came from the target branch.
`flowSpecJsonSchema.json` could be confusing as a developer might think this is for `IFlow` interface, while it's for `IContainer`. How about `floipSpecJsonSchema.json` or directly `IContainer.json` ?